rfcs/template.md

```diff
@@ -0,0 +1,120 @@
+# RFC Title
+
+authors:
+
+- `[Author1](Github Link for Author)`
+
+necessary reviewers:
+
+- [ ] Reviewer 1
+- [ ] Reviewer 2
+- [ ] Reviewer 3
+
+### Introduction
+
+One paragraph explanation of the feature(s), change(s), or addition(s) being
+proposed. A diagram can often be helpful here.
```

Comment:
```suggestion
<!--
One paragraph explanation of the feature(s), change(s), or addition(s) being
proposed. A diagram can often be helpful here.
-->
```

Reply:
Don't think we want this unrendered in the markdown right?

Reply:
Or maybe another way: why _do_ we want this rendered in markdown? It's certainly not an over my dead body, but having this available in your editor but not render if you forget to pull it out of the doc seems useful. But just a suggestion!

Reply:
Ah. I guess I look at it for those reading the template on GH (not yet having pulled it down).